pyne/r2s.py

```diff
@@ -94,6 +94,10 @@ def irradiation_setup(flux_mesh, cell_mats, alara_params, tally_num=4,
 
     if m.structured:
         cell_fracs = discretize_geom(m, num_rays=num_rays, grid=grid)
+        # tag cell fracs
+        use_subvoxel = True
+        if use_subvoxel:
+            m.tag_cell_fracs(cell_fracs)
```

Comment:
These three lines are logically equivalent to only the last line, `m.tag_cell_fracs(cell_fracs)`